axelrod/strategies/mindreader.py

```diff
@@ -91,7 +91,7 @@ class MirrorMindReader(ProtectedMindReader):
     classifier = {
         "memory_depth": float("inf"),
         "stochastic": False,
-        "makes_use_of": set(),
+        "makes_use_of": {"game"},
```

Comment:
Technically this one uses the opponent strategy, so whatever the opponent does determines the classifier.

Reply:
:|  Okay, I'm going to revert this back to set(), but I guess there's really not a correct value here.